src/components/ChatEntry.js

```diff
@@ -1,22 +1,42 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp.js';
 
 const ChatEntry = (props) => {
+  const flipMyHearts = () => {
+    props.heartCallback(props.id);
+  };
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div
+      className={`chat-entry ${
+        props.sender === 'Vladimir' ? 'local' : 'remote'
```

Comment:
Tip: This is kind of brittle since it means we can't re-use these components for a chat between different people.